src/store/modules/DownloadModule.ts

```diff
@@ -109,10 +122,18 @@ export const DownloadModule = {
                 await dispatch('_installModsAndResolveConflicts', { combos: modsWithDependencies, profile, downloadId });
                 commit('setDone', downloadId);
             } catch (e) {
+                const r2Error = R2Error.fromThrownValue(e);
                 if (downloadId) {
                     commit('setFailed', downloadId);
+                    if (game.internalFolderName ===  rootState.activeGame.internalFolderName
```

Comment:
Nitpick: checking the profile path should be enough as it contains the game's name too.